src/WebJobs.Script.WebHost/Extensions/FunctionMetadataExtensions.cs

```diff
@@ -130,13 +137,26 @@ private static async Task<string> GetTestData(string testDataPath, ScriptJobHost
         private static Uri GetFunctionHref(string functionName, string baseUrl) =>
             new Uri($"{baseUrl}/admin/functions/{functionName}");
 
-        private static Uri GetFunctionInvokeUrlTemplate(string baseUrl, string functionName, IWebJobsRouter router)
+        private static Uri GetFunctionInvokeUrlTemplate(string baseUrl, FunctionMetadata functionMetadata, string routePrefix)
         {
-            var template = router?.GetFunctionRouteTemplate(functionName);
-
-            if (template != null)
+            var httpBinding = functionMetadata.InputBindings.FirstOrDefault(p => string.Compare(p.Type, "httpTrigger", StringComparison.OrdinalIgnoreCase) == 0);
```

Comment:
Getting route info from raw json. In all cases (including proxies and precompiled functions) we have this metadata.